src/cipher.rs

```diff
@@ -108,6 +110,45 @@ pub extern "C" fn rustls_certified_key_build(
     }
 }
 
+/// Create a copy of the rustls_certified_key with the given OCSP response data
+/// as DER encoded bytes. The OCSP response may be given as NULL to clear any
+/// possibly present OCSP data from the cloned key.
+/// The cloned key is independant from its original and needs to be freed
+/// by the application.
+#[no_mangle]
+pub extern "C" fn rustls_certified_key_clone_with_ocsp(
+    key: *const rustls_certified_key,
+    ocsp_response: *const rustls_slice_bytes,
+    cloned_key_out: *mut *const rustls_certified_key,
+) -> rustls_result {
+    ffi_panic_boundary! {
+        let cloned_key_out: &mut *const rustls_certified_key = unsafe {
+            match cloned_key_out.as_mut() {
+                Some(c) => c,
+                None => return NullParameter,
+            }
+        };
+        let certified_key: Arc<CertifiedKey> = unsafe {
+            match (key as *const CertifiedKey).as_ref() {
+                Some(c) => arc_with_incref_from_raw(c),
+                None => return NullParameter,
+            }
+        };
+        let mut nkey = certified_key.deref().clone();
```

Comment:
```suggestion
        let mut new_key = certified_key.deref().clone();
```

Let's use a slightly longer name for clarity - at first I thought the `n` in `nkey` was for length.